package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dosomething/gateway",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
@deadlybutter on my local I changed it from `1.2.0` to `1.1.4` but looks like this still thinks its on `1.1.3` - that is what we expected though right?

Reply:
The [master](https://github.com/DoSomething/gateway-js/blob/master/package.json#L3) package.json is on `1.1.3`, so the final change it's comparing is from `1.1.3` to `1.1.4`. In the first commit of the PR I see the `1.2.0` version like you expected though.